src/Block/Service/AbstractBlockService.php

```diff
@@ -63,7 +52,11 @@ public function __construct($name = null, EngineInterface $templating = null)
      */
     public function renderResponse($view, array $parameters = [], Response $response = null)
     {
-        return $this->getTemplating()->renderResponse($view, $parameters, $response);
+        $response = $response ?? new Response();
+
+        $response->setContent($this->getTemplating()->render($view, $parameters));
```

Comment:
setContent returns the response, so you can return directly if you want :P

Reply:
I do not like it =) with these two lines i can set breakpoint on a second line with `return $response` after setContent; with one line it won't be convenient